Libraries/LibGUI/BreadcrumbBar.cpp

```diff
@@ -139,4 +139,10 @@ void BreadcrumbBar::set_selected_segment(Optional<size_t> index)
     segment.button->set_checked(true);
 }
 
+void BreadcrumbBar::doubleclick_event(MouseEvent& event)
+{
+    if (on_doubleclick)
+        on_doubleclick(event);
```

Comment:
Please update the commit message so it matches the new name for this hook :)

Reply:
Whoops 😄 It should match now.